src/onyx/peer/task_lifecycle.clj

```diff
@@ -9,6 +9,7 @@
               [onyx.messaging.acking-daemon :as acker]
               [onyx.peer.pipeline-extensions :as p-ext]
               [onyx.peer.function :as function]
+              [onyx.plugin.core-async]
```

Comment:
Oops. That was just a hack to get it working before I did plugin resolution and it's no longer required.